esp32-s3-pv18-usb.yaml

```diff
@@ -0,0 +1,581 @@
+substitutions:
+  device_name: "pv18_5248_prem"
+  name: "Must inverter"
+  device_description: "Monitor MUST inverter via USB serial"
+  inverter_id: inverter
+  updates: 20s
+  api_key: xxx
+  ota_password: xxx
+
+esphome:
+  name: inverter-monitor
+  comment: "Inverter monitor"
+
+esp32:
+  board: esp32-s3-devkitc-1
+  framework:
+    type: esp-idf
+
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+  # Static IP adress sometimes nedded
+  # use_address: 192.168.x.x
+
+  # Enable fallback hotspot (captive portal) in case wifi connection fails
+  # ap:
+  #   ssid: ${device_name}
+  #   password: !secret ap_password
+
+captive_portal:
+
+# INFO is better for ESP perfomance, change to DEBUG and uncommit hardware_uart if you have any issues
+logger:
+  level: INFO
+  # hardware_uart: uart0
+
+api:
+  encryption:
+    key: ${api_key}
+
+ota:
+  - platform: esphome
+    password: ${ota_password}
+
+web_server:
+  port: 80
+
+switch:
+  - platform: restart
+    name: "${device_name} restart"
+
+# Temprorary external_components
+external_components:
+  - source: github://pr#9425
```

Comment:
Is it necessary? Can’t the original implementation in the current ESPHome version be used?